content/millennium/dstu2/medications/immunization.md

```diff
@@ -9,11 +9,11 @@ title: Immunization | DSTU 2 API
 
 ## Overview
 
-The Immunization resource includes the view of current and historical administration of vaccinations to a patient in all healthcare settings. This resource contains the functionality to query a patient's immunization history.
+The Immunization resource is intended to cover the recording of current and historical administration of vaccines to patients across all healthcare disciplines, in all care settings, and all regions.  This resource contains the functionality to query a patient's immunization history. Detailed administration records may be found in the [`MedicationAdministration`] resource, while this Immunization resource would represent the known vaccination history regardless of where the administration itself was done.
 
-Detailed administration records may be found in MedicationAdministration, while the Immunization resource would represent the known vaccination history regardless of where the administration itself was done.
+An immunization reaction may indicate an allergy or intolerance. If so, a separate [`AllergyIntolerance`] resource instance must be created.
```

Comment:
this is a weird business rule/recommendation that I don't think I've seen elsewhere.

@cldeitrich, thoughts on how to approach this one?

Reply:
looks like this is replicating information from HL7:
https://hl7.org/fhir/DSTU2/immunization.html#bnr